pkg/dashboard/static/list-view.js

```diff
@@ -79,6 +79,20 @@ function buildChartCard(elm) {
 
         loadChartHistory(chart.namespace, chart.name, elm.chart_name)
     })
+
+    // check if upgrade is possible
+    $.getJSON("/api/helm/repositories/latestver?name=" + elm.chartName).fail(function (xhr) {
```

Comment:
I was reviewing the change locally and I observed that it takes a while for the API call to work and then display if the chart is upgradeable and this is happening eachtime we visit the 'Installed chart' page

Can we cache this response so that if the user revists the 'Installed chart' page again then we don't make the API call and rather use cache? 

It will be a better UX. WDYT?

Reply:
I already tested this locally using cache and seems to be working fine
If you agree then I will push the fix

Reply:
With cache, the question is when/how to invalidate that cache. If user updates the repo - the cache has to be cleared.
Feel free  to propose PR with your approach to that

Reply:
Actually, I tested the code using cache and it was working fine. I upgraded and downloaded releases and each time it was showing the correct result.
Let me create a PR and you can review it